### DISCUSSION TITLE
Can I add a Paragraph in the Paragraph with style attributes?

### DISCUSSION THREAD

**Discussion by heijang** [LibrePDF/OpenPDF#901] [Q&A]
When I add a paragraph in the paragraph, some style attributes including alignment are not applied to the paragraph inside.
I need it to use a paragraph with a title and body and use it repeatedly.

```
Paragraph paragraph = new Paragraph();
paragraph.add("OUTER");
paragraph.add(Chunk.NEWLINE);
Paragraph innerParagraph = new Paragraph();
innerParagraph.add("INNER");
innerParagraph.setAlignment(Element.ALIGN_RIGHT);
paragraph.add(innerParagraph);

Document document = new Document();
document.add(paragraph);
```

## Expect
```
OUTER
                                    INNER
```

## Result
```
OUTER
INNER
```

**Comment by asturio** [comment]
In this case you only have one Paragraph (the outer one). When adding the inner paragraph only its contents-Chunks are copied. The alignment is only in the outer Paragraph. Only the alignment of the outer paragraph is counted, when adding it to the document.

You should add the 2 paragraph to the document separately, to achieve what you want.
